internal/config/inputs_test.go

```diff
@@ -92,79 +79,9 @@ func TestInputs_Config_ok(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			diff := deep.Equal(got, want)
-			for _, d := range diff {
-				t.Error(d)
-			}
-		})
-	}
-}
-
-func TestInputs_Config_err(t *testing.T) {
```

Comment:
The CLI itself no longer _requires_ any config, we do have required fields in the `action.yml` though, mostly because requiring them makes the action more likely to be used correctly. (In CI we assume we'll want to build binaries for more than one platform, for example, so requiring the `os` and `arch` inputs ensures that the platform is considered/added to the build matrix.)